inc/class-book.php

```diff
@@ -151,16 +150,21 @@ static function getBookInformation( $id = null, $contributors_as_string = true,
 			$expected_the_content = [ 'pb_custom_copyright', 'pb_about_unlimited' ];
 			$expected_url = [ 'pb_cover_image' ];
 			foreach ( get_post_meta( $meta_post->ID ) as $key => $val ) {
-
 				// Skip anything not prefixed with pb_
-				if ( ! preg_match( '/^pb_/', $key ) ) {
+				if ( 0 !== strpos( $key, 'pb_' ) ) {
 					continue;
 				}
 				// Skip contributor meta (already done, look up)
 				if ( $contributors->isValid( $key ) || $contributors->isDeprecated( $key ) ) {
 					continue;
 				}
 
+				if ( $key === 'pb_institutions' ) {
+					$book_information[ $key ] = $val;
+
+					continue;
+				}
+
```

Comment:
We want to expose the value as an array instead of processing into a comma separated string